_posts/2021-02-00-the-life-science-community.md

```diff
@@ -0,0 +1,193 @@
+
+Ideally these are short term projects, so we can showcase many different applications of Dask. We aim to spend around 20% of project effort on applications.
+
+### How will we know what success looks like?
+The role of Dask Life Science Fellow has a very broad scope, so there are a lot of different ways we could be successful within this space.
+
+Some indicators of success are:
+* Bugs being clearly described, or bottlenecks clearly identified
+* Bug fixes
+* Improvements or new features made to Dask infrastructure
+* Improvements or new features made in related project repositories
+* Better integration or support for Dask made in related project repositories for life sciences
+* Better documentation with examples tailored to specific areas of life science
+* Blogposts written (ideally in collaboration with Dask users)
+* Talks given
+* Webinars produced
+* Tutorials created
+
+We won't have the time or the resources to do all the things, but we will be able to make an impact by focussing on a subset.
+
+## Limitations
+
+The information we discovered talking to the life science community is likely to be biased in a few different ways.
+
+My (Genevieve's) network is strongest among imaging scientists, and among people in Australia. It's much less strong for other fields in life science, as my original training is in physics.
+
+The Dask project has strong links to other open source python projects, including scientific software. The Dask developer community also has strong links from companies including NVIDIA, Quansight, and others. They are likely to be over-represented among the people we spoke to.
+
+It's much harder to find people who aren't using Dask at all yet but have problems that would be a good fit for it. These people are very unlikely to be, say following [Dask on twitter](https://twitter.com/dask_dev/), and probably won't be aware that we're looking for them.
+
+I don't think there are any perfect solutions to these problems.
+We've tried to mitigate these effects by using loose second and third degree connections to spread awareness, as well as posting in science public forums.
+
+## Methods
+
+We used a variety of approaches to gather feedback from the life science community.
+
+* A [short survey](https://t.co/0NeknSdrO9?amp=1) was created to gather comments
+* It was advertised by the [@dask_dev](https://twitter.com/dask_dev/) twitter account
+* We asked related software projects consider retweeting for reach ([example](https://twitter.com/napari_imaging/status/1360090299901505543))
+* We posted in scientific Slack groups and online public forums
+* We emailed other life scientists in our network, asking them to let their networks know too
+* We contacted a number of life science researchers directly.
+* We contacted several other scientific software groups directly and spoke with the developers.
```

Comment:
Putting my marketing hat on, what do we want users to do after reading this blogpost?  What is our _call to action_?

We might, for example, leave them with a set of links to related projects (napari, cell profiler, etc.) and educational resources like examples.dask.org links (maybe the imaging post).

Or we might ask them to get in touch if they have other suggestions, perhaps leaving your form at the bottom here.

Or perhaps something else entirely.

Reply:
Call to action - join the slack life sciences channel